### PR TITLE
feat(device): Server Side Events

### PR DESCRIPTION
**Pull request by AleksaBajat**
**Description**:

Device has the ability to listen to and process SSE in a way that will not block other SSEs to get processed. Closes #40.

**Checklist**:

- [x] Unit tests: Have you added or updated unit tests for your changes and do you need to?
- [x] Integration tests: Have you added or updated integration tests and do you need to?
- [x] Bechmarks: Have you added benchmarks and if so did you run them?
- [x] Documentation: Have you update the relevant documentation to reflect the changes?

### CODE REVIEW

File: android/launcher/lib/src/sse/sse_entry.dart [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+abstract class SSE {}
+
+class SSEUndefinedMessage implements SSE {}
+
+class SSENoopMessage implements SSE {}
+
+void sseEntryPoint(SendPort port) async {
+  const url = String.fromEnvironment("sse_url",
+      defaultValue:
+          "http://10.0.2.2:8000/event/connect?id=1218cbbc-f2d7-4a06-b8d9-d8d6d3732663");
```
Comment (NikolaMilosa) [inline android/launcher/lib/src/sse/sse_entry.dart:27]: Looks super random? This url is kind of a must. If there isn't anything wouldn't it be better to just bail?

**Review by NikolaMilosa — APPROVED** [review]
Things are going the right direction 😄

File: Makefile [hunk 2]
```diff
@@ -59,6 +59,7 @@ flutter-create-emulator: ## Shorthand for setting up an emulator
 	sdkmanager emulator
 	sdkmanager platform-tools
 	avdmanager create avd -n "android_tv" -k $(ANDROID_IMAGE) --force
+	adb reverse tcp:8000 tcp:8000
```
[the remark this answers is not in the document]
Comment (AleksaBajat) [reply Makefile:62]: This is so that the device can connect to the nginx.

File: android/launcher/lib/src/sse/sse_entry.dart (hunk 1 above)
Comment (AleksaBajat) [reply android/launcher/lib/src/sse/sse_entry.dart:27]: It is, will be properly done in the next iter.

**Pull request merged**